### PR TITLE
Fix: Flaky test ProjectSwapDetails.spec

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

ProjectSwapDetails.spec failed because the white space what not the same.

# Changes

* Normalize the white space in the test of ProjectSwapDetails.spec.

# Tests

Only test changes

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/lib/components/project-detail/ProjectSwapDetails.spec.ts [hunk 1]
```diff
@@ -80,7 +81,9 @@ describe("ProjectSwapDetails", () => {
       swapCommitment: mockSnsFullProject.swapCommitment as SnsSwapCommitment,
     });
 
-    expect(await po.getSaleEnd()).toEqual("Oct 4, 2023 3:00 PM");
+    expect(normalizeWhitespace(await po.getSaleEnd())).toEqual(
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/project-detail/ProjectSwapDetails.spec.ts:84]: Some getters in `ProjectSwapDetailsPo` already do `trim()` on the result before returning.
Do you think it makes sense for `normalizeWhitespace` to go inside `getSaleEnd()`?

Also, as far as I remember, the kind of whitespace used only depends on the Node version.
Have you seen this fail on CI or are you maybe using a different Node version when you see this fail?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/tests/lib/components/project-detail/ProjectSwapDetails.spec.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ describe("ProjectSwapDetails", () => {
       swapCommitment: mockSnsFullProject.swapCommitment as SnsSwapCommitment,
     });
 
-    expect(await po.getSaleEnd()).toEqual("Oct 4, 2023 3:00 PM");
+    expect(await po.getSaleEnd()).toEqual("Oct 4, 2023 3:00 PM");
```
Comment (lmuntaner) [inline frontend/src/tests/lib/components/project-detail/ProjectSwapDetails.spec.ts:83]: The change is using a normal whitespace

File: frontend/src/tests/lib/components/project-detail/ProjectSwapDetails.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/components/project-detail/ProjectSwapDetails.spec.ts]: done

**Review by dskloetd — APPROVED** [review]
Thanks!

**Pull request merged**